Source/Core/Core/State.cpp

```diff
@@ -64,7 +64,7 @@ static Common::Event g_compressAndDumpStateSyncEvent;
 static std::thread g_save_thread;
 
 // Don't forget to increase this after doing changes on the savestate system
-static const u32 STATE_VERSION = 38;
+static const u32 STATE_VERSION = 40;
```

Comment:
Should I increment STATE_VERSION twice like this because two commits introduce savestate differences, or should it only be once for the entire PR?